src/jit/lower.cpp

```diff
@@ -1546,7 +1546,7 @@ void Lowering::LowerCall(GenTree* node)
             }
 #endif
 #if defined(_TARGET_AMD64_) && defined(FEATURE_READYTORUN_COMPILER)
-            if (call->gtEntryPoint.addr != nullptr)
```

Comment:
It would be nice to refactor the code to get rid of this ifdef completely. I may tackle it in separate commit.